docs/mobile/game-session/game_session_db_design.md

```diff
@@ -0,0 +1,53 @@
+# DynamoDB Design
+
+## Entities
+
+### Config
+
+This entity contains the global settings and configurable parts of the application. The list of items included here is as follow
+
+- Points granted to each team estimated the right popular trick answer
+- Points granted per to teams guessed the right answer
+
+### GameSession
+
+The higher level entity contains all information related to a game from start to finish.
+
+- Game: A game object that comes from Aurora. It's a snapshot of the data when the game starts
+- Teams: An array of team entities that involved in the game
+- StartedDateTimeUTC: Date time of when the game started in UTC
+- MultiDeviceMode: Boolean indicating if it's a multi-device with facilitator or not
+
+### Team
+
+- TeamNo / TeamId: Numeric unique number assigned to the team
+- Team name: Name chosen to show fo the team. This helps for future substituting `team 1` string with chosen team names
+- Question: The chosen question for the team
+- GameCode: A 4-digit code to enter to join the team
+
+### TeamMember
+
+- TeamId: The teamId this member is a member of
+- DeviceId: Unique number associated to the device
+- IsFacilitator: A boolean indicating if this device is the facilitator in multi-device mode
+
+### Question
+
+- QuestionId: Unique number for each question
+- Question: The text for the question
+- Hints: an array of hints for the question
+- ImageURL: URL of the image associated to this question
+- Answer: The right answer for this question
+
+### TeamAnswer
+
+- AnswerId: A unique id for each answer
+- Text: The answer chosen by a team member
+- TeamMember: DeviceId entered the answer
+- IsTrickAnswer: Indicates if this answer is chosen to be a trick answer
+
+### TeamRoundResults
+
+- TeamId: The unique number identifies the team choosing the answer
+- AnswerId: Identifies which answer
+- QuestionTeamID: Identifies which team's question is this
```

Comment:
Might it make sense to further differentiate TeamId from QuestionTeamId -- e.g., perhaps updating the former to AnswerTeamId? For consistency, perhaps QuestionTeamID might be tweaked to QuestionTeamId.

Reply:
Following up with Sinclair's comment, it may make consistent sense to update AnswerId to AnswerTeamId as well as the formers if we're looking to distinguish them. Otherwise, I think it makes sense design wise.